brands/shop/furniture.json

```diff
@@ -354,6 +374,14 @@
       "shop": "furniture"
     }
   },
+  "shop/furniture|Mömax": {
```

Comment:
Without a wikidata value this one won't actually show up in editors.

Reply:
Okay, thank you. The problem is, that (like _XXXLutz_) it belongs to the company "Möbelix". It has in fact got two Wikidata pages - one individual one with barely any data and a redirect to "Möbelix" (in the same way as _XXXLutz_).